src/arr-crud.js

```diff
@@ -0,0 +1,63 @@
+import { dragndrop, renderTasks } from './drag-and-drop.js'; // eslint-disable-line
+import { checkboxesUpdate } from './status-updates.js'; // eslint-disable-line
+import { setStorage } from './index.js'; // eslint-disable-line
+import { seeMoreUpdate } from './status-updates.js'; // eslint-disable-line
+
+export class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export class ToDoList {
+  constructor(toDoArr = []) {
+    this.tasksArr = toDoArr;
+  }
+
+  addTask(taskobj) {
+    this.tasksArr.push(taskobj);
+    return this.tasksArr;
+  }
+
+  removeTask(targetindex) {
+    const idx = targetindex - 1;
+    this.tasksArr = this.tasksArr.filter((taskobj, index) => index !== idx);
+    for (let x = 0; x < this.tasksArr.length; x += 1) {
+      this.tasksArr[x].index = x + 1;
+    }
+    return this.tasksArr;
+  }
+
+  clearAll() {
+    this.tasksArr = [];
+  }
```

Comment:
- [ ]  The function for todolist to add and remove, it needs to target the localStorage